src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Metric/LocalizableFilterIntegration.php

```diff
@@ -21,6 +21,8 @@ protected function setUp()
         parent::setUp();
 
         if (1 === self::$count || $this->getConfiguration()->isDatabasePurgedForEachTest()) {
+            $this->resetIndex();
```

Comment:
Maybe not in the `TestCase`, but at least in `AbstractProductQueryBuilderTestCase`.